### DISCUSSION TITLE
PEP 730 – Adding iOS as a supported platform

### DISCUSSION THREAD

**Discussion by kkebo** [holzschu/a-shell#788] [General]
https://peps.python.org/pep-0730/

CPython 3.13 appears to officially support iOS. This possibly makes building CPython for a-Shell easier.

**Comment by holzschu** [comment]
Thank you very much for the link. 
I even had the opportunity to add my voice in support to this PEP. I'll see exactly exactly how much easier it makes building CPython when I move to CPython 3.13 (tentatively scheduled for August-September 2024). It will require changes in all the existing packages, because it changes the way Python sees iOS from "Darwin" to "iOS". On the plus side, it will remove the current confusion between "darwin-arm64" (OSX running on an M-series processor) and "darwin-arm64" (iOS running on a recent iPad or iPhone).

**Reply by kkebo** [reply]
Hmm, it sounds like a right direction. Thank you for sharing the further information. I am hoping that once all packages have been migrated, it will be easier to keep up with Python versions (e.g. 3.14, 3.15, and so on).
